tests/e2e/set-ffi-env-e2e

```diff
@@ -230,6 +240,24 @@ EOF
   systemctl restart qm
 }
 
+install_autosd_repo() {
+    local release_version_id
+    local base_arch
+    local autosd_repo_filepath
+    local autosd_repo_base_url
+    release_version_id=$(grep ^VERSION_ID= /etc/os-release | cut -d = -f 2 | cut -d \" -f 2)
+    base_arch=$(arch)
```

Comment:
Please use this as template, 
I assume it is more clear
https://github.com/containers/qm/blob/main/setup#L160-L166

Please use the dnf variables